scripts/eval/eval.py

```diff
@@ -56,6 +56,15 @@ def load_model(model_cfg: DictConfig, tokenizer: PreTrainedTokenizerBase,
     return composer_model
 
 
+def log_analytics_details(mosaicml_logger: MosaicMLLogger, model_name: str):
+    mosaicml_logger.log_metrics({
+        'uses_llmfoundry': True,
+        'model_name': model_name,
+        'llmfoundry_run_type': 'Eval',
```

Comment:
Could all of the keys be prefixed with something like `mosaicml/llmfoundry/`? Any other suggestions here? 

Eg etas are logged as `mosaicml/time/remaining_estimate`

Reply:
added `llmfoundry/` as a prefix b/c the mosaicml logger already prefixes w/ `mosaicml`